source/normalize-arguments.js

```diff
@@ -51,6 +51,16 @@ module.exports = (url, options, defaults) => {
 		...options
 	};
 
+	const {baseUrl} = options;
+	Object.defineProperty(options, 'baseUrl', {
+		set: _ => {
```

Comment:
`_` => `()`

Reply:
Then node will throw an error that setter is missing an argument, because how you can set something to somewhat without knowing the value?

Reply:
Weird. Usually it threw an error that setter is missing an argument. I'll change it :)